packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -176,12 +204,12 @@ export async function wrapRunMigration(
   exec: (context: MigrationContext) => void
 ): Promise<void> {
   try {
-    sendTelemetryEvent(Component.core, TelemetryEvent.ProjectMigratorMigrateStartV3);
+    sendTelemetryEvent(Component.core, TelemetryEvent.ProjectMigratorMigrateStart);
```

Comment:
Are we reusing previous migration event? If yes, can we add a property to distinguish 3.0->4.0 upgrade, and 4.0->5.0 upgrade?

Reply:
OK. will use another pr for the telemetry property